beamer/tests/agent/unit/utils.py

```diff
@@ -148,6 +148,8 @@ def make_context() -> Tuple[Context, Config]:
     context = Context(
         requests=Tracker(),
         claims=Tracker(),
+        source_chain_id=SOURCE_CHAIN_ID,
+        target_chain_id=TARGET_CHAIN_ID,
```

Comment:
Why do we need this in `Context`?

Reply:
We don't need it in the current state. But we should also consider other events which are irrelevant for the agent.
For example, `RequestFilled` event is irrelevant for the agent if the source chain id doesnt match the agent's source chain id.
I haven't thought through all events for now.

Reply:
By comparing the source chain id of the RequestFilled event with the agent's source chain id we know that the agent is not interested in this event